vital/tests/rpc_reader.h

```diff
@@ -79,6 +80,8 @@ kwiver::vital::simple_camera_rpc read_rpc( std::string filename )
           case 7:
             image_offset( word_idx ) = value;
             break;
+          case 8:
+            image_dimension( word_idx ) = value;
```

Comment:
we might want a static cast here since `value` is `double` but `image_dimension` contains `int`

Reply:
done